src/quo2/components/context_tags.cljs

```diff
@@ -47,8 +48,10 @@
       (trim-public-key public-key)]]))
 
 (defn user-avatar-tag [_ _]
-  (fn [params public-key]
-    (let [{:keys [name photo]} @(re-frame/subscribe [:contacts/name-and-photo public-key])]
+  (fn [params _]
+    (let [{:keys [names public-key]} @(re-frame/subscribe [:multiaccount/contact])
+          {:keys [nickname three-words-name]} names
```

Comment:
As components should be abstract without any reframe calls,
should we move this subscription to `quo2/screens/context_tags.cljs:51`?

Reply:
i think even in screens we shouldn't use re-frame, there must be just mock data